src/tds/sec_negotiate_openssl.h

```diff
@@ -37,6 +37,11 @@
 #error HAVE_OPENSSL not defines, this file should not be included
 #endif
 
+#if defined(LIBRESSL_VERSION_NUMBER) && LIBRESSL_VERSION_NUMBER < 0x2070000fL
+#undef OPENSSL_VERSION_NUMBER
+#define OPENSSL_VERSION_NUMBER 0x1000107fL
+#endif
+
```

Comment:
looks like these lines are reused in the below file, should not they be in an header, like freetds/tls.h?

Reply:
That sounds correct... Let me fix the patch